src/v1/sources/shopify/pixelTransform.js

```diff
@@ -77,6 +77,8 @@ function processPixelEvent(inputEvent) {
     version: '2.0.0',
   });
   message.setProperty('context.topic', name);
+  message.setProperty('context.shopifyDetails', data);
```

Comment:
what about 'context.shopifyDetails' for server-side events ?

Reply:
Added support for server-side events for the pixel app.

Reply:
reverting these after discussion

Reply:
this will impact tracker app server-side events too, so not taking it currently